src/index.html

```diff
@@ -1,15 +1,145 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post" onsubmit="onSubmit()">
+      <fieldset class="block">
+        <legend>Personal information
+        </legend>
+        <div class="input">
+          <label for="surname">Surname:
+            <input type="text"
+            name="surname" id="surname">
+          </label>
+        </div>
+        <div class="input">
+          <label for="name">Name:
+            <input type="text"
+            name="name" id="name">
+          </label>
+        </div>
+        <div class="input">
+          <label for="age">How old are You?
+            <input type="number"
+            name="age" id="age"
+            value="12"
+            min="12" max="100">
+          </label>
+        </div>
+        <div class="input">
+          <label for="date">Full date of birth:
+            <input type="date"
+            name="date" id="date">
+          </label>
+        </div>
+        <div class="input">
+          <label for="agreement">I accept the term of the agreement
+            <input type="checkbox"
+            name="agreement" id="agreement"
+            required>
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="block">
+        <legend>Registration
+        </legend>
```

Comment:
повиправляй всі `legend`
```suggestion
        <legend>
          Registration
        </legend>
```